src/assets/questionsAndAnswers/questionsAndAnswers.js

```diff
@@ -144,6 +144,15 @@ export default {
                     },
                     {
                         "id": "useQuestion2",
+                        "question": "What does “Concept Map” mean, and why aren’t these data available here for decision making? ",
+                        "answer": "The data presented within the IWAAs water availability series of maps demonstrate a new USGS capability for calculating near real-time estimates of water budget variables.  The process (or concept) behind the data acquisition, modeling, and delivery of water temperature or water storage data throughout the lower 48 states are orchestrated using an operational pipeline in the Cloud (See About tab for more information).  While the models used for these calculations are complex and verified (" +
+                        "<a href='https://doi.org/10.5066/P9NM8K8W' " + 
```

Comment:
I think we would also want to make sure we add references in the text to the references on the bottom?

Reply:
That makes sense